examples/resourceServer/server/rest.js

```diff
@@ -16,6 +16,8 @@ JsonRoutes.Middleware.use(
 );
 
 /**
+ * OAUTH FLOW - A7.2
```

Comment:
That's the idea.  I'm trying to go through the whole thing and match up code with diagram, and vice-versa.